src/request/payload.rs

```diff
@@ -8,18 +8,18 @@ use erased_serde::Serialize;
 
 /// The data and options for a push notification.
 #[derive(Debug, Clone)]
-pub struct Payload {
+pub struct Payload<'a, 'b> {
```

Comment:
Why do we need two different lifetimes, one for `device_token` and another for `NotificationOptions`? Wouldn't only `'a` be enough, giving a guarantee that the token and options both live longer than the `Payload`?

Reply:
Because - as far as I know - then both lifetimes must be the same, meaning that you can't use a 'static lifetime for one reference and a locally scoped lifetime for the other reference...